src/services/therapists.ts

```diff
@@ -0,0 +1,27 @@
+import { Op } from 'sequelize';
+import { Therapist, User } from '../models';
+
+const getAllTherapist = async (name: string, page: number, limit: number) => {
```

Comment:
we don't neet to have limit here as a parameter, Insted let us import `Limit` and use it Here